### PR TITLE
Add RTL support

### PR DESCRIPTION
**Pull request by Gilad-Shnoor**

### CODE REVIEW

File: src/Calendar.tsx [hunk 1]
```diff
@@ -123,8 +125,9 @@ export const Calendar = React.memo(
       cellHeight,
       dateRange,
       style,
+      isRTL,
     }
-
+
```
Comment (acro5piano) [inline src/Calendar.tsx:130]: Can you remove this?

**Review by acro5piano — CHANGES_REQUESTED** [review]
Thank you for your contribution. I love it.
In addtion, 

- Can you add `isRTL` to the API documentation?

https://github.com/acro5piano/react-native-big-calendar#api-documentation

- Can you add a story for describing RTL support?

https://github.com/acro5piano/react-native-big-calendar/blob/master/stories/desktop.stories.tsx

Thanks very much!

**Comment by Gilad-Shnoor** [comment]
done

**Review by acro5piano — APPROVED** [review]
Thanks!!

**Comment by acro5piano** [comment]
@Gilad-Shnoor Thank you for your contribution!

**Comment by acro5piano** [comment]
Released in v0.7.0

https://github.com/acro5piano/react-native-big-calendar/releases/tag/v0.7.0

**Pull request merged**